x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -67,119 +63,88 @@ export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) =>
     </EuiButtonEmpty>
   );
 
+  const showConfirmDeleteModal = () => {
+    setShowConfirmDelete(true);
+    setIsPopOverOpen(false);
+  };
+
   return (
-    <EuiPanel color="subdued">
-      <EuiFlexGroup alignItems="center">
-        <EuiFlexItem>
-          <EuiFlexGroup direction="column" gutterSize="xs">
+    <>
+      <EuiPopover
+        button={actionButton}
+        isOpen={isPopOverOpen}
+        closePopover={() => setIsPopOverOpen(false)}
+      >
+        <EuiFlexGroup direction="column" gutterSize="none">
+          {pipeline.modelState === TrainedModelState.NotDeployed && (
             <EuiFlexItem>
-              <EuiFlexGroup alignItems="center">
-                <EuiFlexItem>
-                  <EuiTitle size="xs">
-                    <h4>{pipelineName ?? modelTitle}</h4>
-                  </EuiTitle>
-                </EuiFlexItem>
-                <EuiFlexItem grow={false} />
-              </EuiFlexGroup>
-            </EuiFlexItem>
-            {modelTitle && (
-              <EuiFlexItem>
-                <EuiFlexGroup gutterSize="s">
-                  <EuiFlexItem grow={false}>
-                    <EuiTextColor color="subdued">{modelId}</EuiTextColor>
-                  </EuiFlexItem>
-                  <EuiFlexItem>
-                    <span>
-                      <MLModelTypeBadge type={modelType} />
-                    </span>
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </EuiFlexItem>
-            )}
-          </EuiFlexGroup>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiPopover
-            button={actionButton}
-            isOpen={isPopOverOpen}
-            closePopover={() => setIsPopOverOpen(false)}
-          >
-            {pipeline.modelState === TrainedModelState.NotDeployed && (
-              <EuiFlexItem grow={false} style={{ paddingRight: '1rem' }}>
-                <EuiToolTip
-                  position="top"
-                  content={i18n.translate(
+              <div>
```

Comment:
nitpick: Why is this button wrapped in a div? If it's just to restrict the width, we prefer `span` over `div` because span is in-line and div is block. Same with the other div elements in here (in general, if you're using a div in Kibana, that's a signal you're doing something wrong).